Index: Makefile
===================================================================
RCS file: /cvs/ports/devel/quirks/Makefile,v
retrieving revision 1.333
diff -u -p -u -r1.333 Makefile
--- Makefile	11 Jan 2016 15:51:18 -0000	1.333
+++ Makefile	15 Jan 2016 13:56:48 -0000
@@ -5,7 +5,7 @@ CATEGORIES =	devel databases
 DISTFILES =
 
 # API.rev
-PKGNAME =	quirks-2.178
+PKGNAME =	quirks-2.179
 PKG_ARCH =	*
 MAINTAINER =	Marc Espie <espie@openbsd.org>
 
Index: files/Quirks.pm
===================================================================
RCS file: /cvs/ports/devel/quirks/files/Quirks.pm,v
retrieving revision 1.343
diff -u -p -u -r1.343 Quirks.pm
--- files/Quirks.pm	11 Jan 2016 15:51:18 -0000	1.343
+++ files/Quirks.pm	15 Jan 2016 13:56:48 -0000
@@ -389,6 +389,8 @@ my $stem_extensions = {
 	'tesseract-dan-frak' => 'tesseract-dan_frak',
 	'grive' => 'grive2',
 	'py-reportlab-renderPM' => 'py-reportlab',
+	'cmus-flac' => 'cmus',
+	'cmus-wavpack' => 'cmus',
 };
 
 # reasons for obsolete packages
