Index: Makefile
===================================================================
RCS file: /cvs/ports/devel/luastdlib/Makefile,v
retrieving revision 1.12
diff -u -p -u -r1.12 Makefile
--- Makefile	11 Mar 2013 10:50:11 -0000	1.12
+++ Makefile	15 Jan 2016 15:00:36 -0000
@@ -2,27 +2,29 @@
 
 PKG_ARCH =      *
 
-COMMENT=	lua library of common programming tasks
-DISTNAME=	stdlib-20
-PKGNAME=	lua${DISTNAME}
-REVISION=	1
-CATEGORIES=	devel
-
-MASTER_SITES=	http://luaforge.net/frs/download.php/4805/
-HOMEPAGE=	http://luaforge.net/projects/stdlib/
-EXTRACT_SUFX=	.zip
+COMMENT =	lua library of common programming tasks
+V =		41.2.0
+
+GH_ACCOUNT =	lua-stdlib
+GH_PROJECT =	lua-stdlib
+GH_TAGNAME = 	release-v${V}
+
+DISTNAME =	${GH_PROJECT:S/-//}-${V}
+
+CATEGORIES =	devel
+
+HOMEPAGE =	http://lua-stdlib.github.io/lua-stdlib
+
+MAINTAINER =	Florian Stinglmayr <florian@n0la.org>
 
 # MIT
-PERMIT_PACKAGE_CDROM=	Yes
+PERMIT_PACKAGE_CDROM =	Yes
+
+MODULES =	lang/lua
+NO_TEST =	Yes
 
-MODULES=	lang/lua
-NO_BUILD=	Yes
-NO_TEST=	Yes
-WRKDIST=	${WRKDIR}/stdlib
-
-do-install:
-	${INSTALL_DATA_DIR} ${MODLUA_DATADIR} ${MODLUA_DOCDIR}
-	${INSTALL_DATA} ${WRKSRC}/modules/*.lua ${MODLUA_DATADIR}
-	${INSTALL_DATA} ${WRKSRC}/modules/*.html ${MODLUA_DOCDIR}
+CONFIGURE_ENV =		LUA=lua${MODLUA_VERSION:S/.//}
+CONFIGURE_ARGS =	--docdir="${PREFIX}/share/doc/${FULLPKGNAME}"
+CONFIGURE_STYLE =	gnu
 
 .include <bsd.port.mk>
Index: distinfo
===================================================================
RCS file: /cvs/ports/devel/luastdlib/distinfo,v
retrieving revision 1.4
diff -u -p -u -r1.4 distinfo
--- distinfo	18 Jan 2015 03:13:18 -0000	1.4
+++ distinfo	15 Jan 2016 15:00:36 -0000
@@ -1,2 +1,2 @@
-SHA256 (stdlib-20.zip) = DPa43S8P60DviCsQQt5anW2xPQpRaaJMqHq9uWbg+38=
-SIZE (stdlib-20.zip) = 48827
+SHA256 (luastdlib-41.2.0.tar.gz) = ydnRHc0uPpVuaX9UVbX2K6rfIz2HJtgFFNBDInPMJ+k=
+SIZE (luastdlib-41.2.0.tar.gz) = 398390
Index: patches/patch-modules_getopt_lua
===================================================================
RCS file: patches/patch-modules_getopt_lua
diff -N patches/patch-modules_getopt_lua
--- patches/patch-modules_getopt_lua	9 Sep 2009 18:31:33 -0000	1.1
+++ /dev/null	1 Jan 1970 00:00:00 -0000
@@ -1,11 +0,0 @@
-$OpenBSD: patch-modules_getopt_lua,v 1.1 2009/09/09 18:31:33 jolan Exp $
---- modules/getopt.lua.orig	Wed Feb 18 17:28:54 2009
-+++ modules/getopt.lua	Wed Sep  9 13:28:20 2009
-@@ -88,6 +86,7 @@ function getOpt (argIn, options)
-       parseOpt (opt, arg)
-     end
-   end
-+  argOut.n = #argOut
-   return argOut, optOut, errors
- end
- 
Index: pkg/PLIST
===================================================================
RCS file: /cvs/ports/devel/luastdlib/pkg/PLIST,v
retrieving revision 1.5
diff -u -p -u -r1.5 PLIST
--- pkg/PLIST	15 May 2012 15:16:29 -0000	1.5
+++ pkg/PLIST	15 Jan 2016 15:00:36 -0000
@@ -1,44 +1,44 @@
 @comment $OpenBSD: PLIST,v 1.5 2012/05/15 15:16:29 jasper Exp $
 share/doc/${FULLPKGNAME}/
-share/doc/${FULLPKGNAME}/base.html
-share/doc/${FULLPKGNAME}/debug_ext.html
-share/doc/${FULLPKGNAME}/debug_init.html
-share/doc/${FULLPKGNAME}/getopt.html
-share/doc/${FULLPKGNAME}/io_ext.html
-share/doc/${FULLPKGNAME}/lcs.html
-share/doc/${FULLPKGNAME}/list.html
-share/doc/${FULLPKGNAME}/math_ext.html
-share/doc/${FULLPKGNAME}/mbox.html
-share/doc/${FULLPKGNAME}/modules.html
-share/doc/${FULLPKGNAME}/object.html
-share/doc/${FULLPKGNAME}/package_ext.html
-share/doc/${FULLPKGNAME}/parser.html
-share/doc/${FULLPKGNAME}/posix_ext.html
-share/doc/${FULLPKGNAME}/set.html
-share/doc/${FULLPKGNAME}/std.html
-share/doc/${FULLPKGNAME}/strict.html
-share/doc/${FULLPKGNAME}/string_ext.html
-share/doc/${FULLPKGNAME}/table_ext.html
-share/doc/${FULLPKGNAME}/tree.html
-share/doc/${FULLPKGNAME}/xml.html
-share/lua/${MODLUA_VERSION}/base.lua
-share/lua/${MODLUA_VERSION}/debug_ext.lua
-share/lua/${MODLUA_VERSION}/debug_init.lua
-share/lua/${MODLUA_VERSION}/getopt.lua
-share/lua/${MODLUA_VERSION}/io_ext.lua
-share/lua/${MODLUA_VERSION}/lcs.lua
-share/lua/${MODLUA_VERSION}/list.lua
-share/lua/${MODLUA_VERSION}/math_ext.lua
-share/lua/${MODLUA_VERSION}/mbox.lua
-share/lua/${MODLUA_VERSION}/modules.lua
-share/lua/${MODLUA_VERSION}/object.lua
-share/lua/${MODLUA_VERSION}/package_ext.lua
-share/lua/${MODLUA_VERSION}/parser.lua
-share/lua/${MODLUA_VERSION}/posix_ext.lua
-share/lua/${MODLUA_VERSION}/set.lua
+share/doc/${FULLPKGNAME}/classes/
+share/doc/${FULLPKGNAME}/classes/std.container.html
+share/doc/${FULLPKGNAME}/classes/std.list.html
+share/doc/${FULLPKGNAME}/classes/std.object.html
+share/doc/${FULLPKGNAME}/classes/std.optparse.html
+share/doc/${FULLPKGNAME}/classes/std.set.html
+share/doc/${FULLPKGNAME}/classes/std.strbuf.html
+share/doc/${FULLPKGNAME}/classes/std.tree.html
+share/doc/${FULLPKGNAME}/index.html
+share/doc/${FULLPKGNAME}/ldoc.css
+share/doc/${FULLPKGNAME}/modules/
+share/doc/${FULLPKGNAME}/modules/std.debug.html
+share/doc/${FULLPKGNAME}/modules/std.functional.html
+share/doc/${FULLPKGNAME}/modules/std.html
+share/doc/${FULLPKGNAME}/modules/std.io.html
+share/doc/${FULLPKGNAME}/modules/std.math.html
+share/doc/${FULLPKGNAME}/modules/std.operator.html
+share/doc/${FULLPKGNAME}/modules/std.package.html
+share/doc/${FULLPKGNAME}/modules/std.strict.html
+share/doc/${FULLPKGNAME}/modules/std.string.html
+share/doc/${FULLPKGNAME}/modules/std.table.html
+share/lua/${MODLUA_VERSION}/std/
 share/lua/${MODLUA_VERSION}/std.lua
-share/lua/${MODLUA_VERSION}/strict.lua
-share/lua/${MODLUA_VERSION}/string_ext.lua
-share/lua/${MODLUA_VERSION}/table_ext.lua
-share/lua/${MODLUA_VERSION}/tree.lua
-share/lua/${MODLUA_VERSION}/xml.lua
+share/lua/${MODLUA_VERSION}/std/base.lua
+share/lua/${MODLUA_VERSION}/std/container.lua
+share/lua/${MODLUA_VERSION}/std/debug.lua
+share/lua/${MODLUA_VERSION}/std/debug_init/
+share/lua/${MODLUA_VERSION}/std/debug_init/init.lua
+share/lua/${MODLUA_VERSION}/std/functional.lua
+share/lua/${MODLUA_VERSION}/std/io.lua
+share/lua/${MODLUA_VERSION}/std/list.lua
+share/lua/${MODLUA_VERSION}/std/math.lua
+share/lua/${MODLUA_VERSION}/std/object.lua
+share/lua/${MODLUA_VERSION}/std/operator.lua
+share/lua/${MODLUA_VERSION}/std/optparse.lua
+share/lua/${MODLUA_VERSION}/std/package.lua
+share/lua/${MODLUA_VERSION}/std/set.lua
+share/lua/${MODLUA_VERSION}/std/strbuf.lua
+share/lua/${MODLUA_VERSION}/std/strict.lua
+share/lua/${MODLUA_VERSION}/std/string.lua
+share/lua/${MODLUA_VERSION}/std/table.lua
+share/lua/${MODLUA_VERSION}/std/tree.lua
