Index: Makefile
===================================================================
RCS file: /cvs/ports/net/luasocket/Makefile,v
retrieving revision 1.31
diff -u -p -u -r1.31 Makefile
--- Makefile	5 Apr 2015 13:31:14 -0000	1.31
+++ Makefile	3 Apr 2016 18:26:55 -0000
@@ -7,7 +7,7 @@ V=		3.0-rc1
 GH_ACCOUNT=	diegonehab
 GH_PROJECT=	luasocket
 GH_TAGNAME=	v$V
-REVISION=	0
+REVISION=	1
 DISTNAME=	${GH_PROJECT}-$V
 PKGNAME=	${DISTNAME:S/-rc/rc/}
 CATEGORIES=	net
@@ -27,7 +27,8 @@ MAKE_FILE=	makefile
 MAKE_FLAGS=	CC_linux=${CC} \
 		LD_linux=${CC} \
 		CFLAGS_linux="${CFLAGS} -I${MODLUA_INCL_DIR} -fPIC \
-			      -DPIC -DUNIX_HAS_SUN_LEN" \
+			      -DPIC -DUNIX_HAS_SUN_LEN \
+			      -DLUA_COMPAT_APIINTCASTS" \
 		LDFLAGS_linux="${LDFLAGS} -shared -fPIC -o "
 
 do-install:
